packages/salesforce-adapter/src/fetch_profile/fetch_profile.ts

```diff
@@ -80,6 +81,7 @@ export const buildFetchProfile = ({
     metadataQuery,
     maxItemsInRetrieveRequest,
     importantValues: mergeWithDefaultImportantValues(additionalImportantValues),
+    optionalDefaults: optionalDefaults ?? {},
```

Comment:
Receive **extraDependenciesChunkSize**  and expose it through the fetchProfile directly.
```suggestion
    extraDependenciesChunkSize,
```